src/sentry/static/sentry/app/utils/performance/quickTrace/types.ts

```diff
@@ -19,18 +19,31 @@ export type EventLite = {
   parent_span_id: string | null;
 };
 
+export type TraceError = {
+  issue: string;
+  event_id: string;
+  span: string;
+  transaction: string;
+  project_slug: string;
+};
+
 export type TraceLite = EventLite[];
 
 /**
  * The `events-trace` endpoint returns a tree structure that gives
  * the parent-child relationships between events.
  */
-export type TraceFull = Omit<EventLite, 'generation'> & {
+export type QuickTraceEvent = EventLite & {
+  errors?: TraceError[];
+};
+
+export type TraceFull = Omit<QuickTraceEvent, 'generation' | 'children' | 'errors'> & {
```

Comment:
I don't think `'children'` is necessary here since `QuickTraceEvent` does not have a `children` prop. The reason we add `generation` and `errors` here because they exist as optional on `QuickTraceEvent` and we want to make them required.

Reply:
Ah yup, missed this when I removed `children` thanks :+1: